src/components/CaseEditor/CaseEditor.js

```diff
@@ -67,15 +67,42 @@ class _CaseEditor extends Component {
   constructor(props) {
     super(props);
     this.makeLead = this.makeLead.bind(this);
+    this.handleNewImg = this.handleNewImg.bind(this);
+    this.deleteImg = this.deleteImg.bind(this);
+    this.deleteLead = this.deleteLead.bind(this);
     this.state = {
-      lead: ""
+      lead: "",
+      newImg: false,
+      delImg: false
     };
   }
 
   makeLead(src) {
     this.setState({ lead: src });
   }
 
+  handleNewImg(img) {
+    this.setState({ newImg: true });
+    let currentImgs = this.props.case.other_images.length
+    this.props.case.other_images[currentImgs] = {url:img};
+  }
+
+  deleteImg(photo) {
```

Comment:
If I go to a case with one image, add another image, and then delete the first image, then the new image isn't marked as 'lead'.  I think we should always ensure that if there's an image, there's a lead image.